gulpfile.js

```diff
@@ -9,20 +9,26 @@ var PATHS = {
 };
 
 gulp.task('serve', ['nodemon', 'sass'], function() {
+  browserSync.init({
+    proxy: 'localhost:4200'
+  });
 
-    browserSync.init({
-      proxy: 'localhost:4200'
-    });
+  gulp.watch(PATHS.sass, ['sass']);
+  gulp.watch(PATHS.html).on('change', browserSync.reload);
+});
 
-    gulp.watch(PATHS.sass, ['sass']);
-    gulp.watch(PATHS.html).on('change', browserSync.reload);
+gulp.task('production', function(){
+  gulp.src(PATHS.sass)
+    .pipe(sass())
+    .pipe(gulp.dest("dist"));
+  require('./src/server.js');
```

Comment:
In production, a node server should be treated as a [daemon](http://www.linfo.org/daemon.html), we want it to run continuously in the background while having control over it. Thus, to start the server, we do not simply run `node server.js`, instead, we write a systemd script or use tools like [foreverjs](https://github.com/foreverjs/forever).

I suggest you to remove `require('./src/server.js')` in `gulpfile.js` and add a start script in `package.json`

```json
"scripts": {
  "start": "gulp serve"
  "start:prod": "sudo systemctl start sexxi.service"
}
```